a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -765,6 +758,14 @@ <h5>Display statements</h5>
 								content can be read as read aloud speech or
 								dynamic braille</p>
 						</dd>
+												<dt>If text alternatives are provided:</dt>
+						<dd>
```

Comment:
George, this goes at the bottom of this entire block of If/elseif/else statements. (Thats what I did and had to change it)